Structure_Engine/Create/SectionProperties/ConcreteSection.cs

```diff
@@ -98,8 +100,9 @@ public static ConcreteSection ConcreteFreeFormSection(List<ICurve> edges, Concre
 
         [Description("Generates a concrete section based on a Profile and a material. \n This is the main create method for concrete sections, responsible for calculating section constants etc. and is being called from all other create methods for concrete sections.")]
         [Input("profile", "The section profile the concrete section. All section constants are derived based on the dimensions of this.")]
-        [Input("material", "concrete material to be used on the section. If null a default material will be extracted from the database.")]
-        [Input("name", "Name of the concrete section. If null or empty the name of the profile will be used.")]
+        [Input("material", "concrete material to be applied to the section. If null a default material will be extracted from the database.")]
+        [Input("name", "Name of the concrete section. If null or empty the name of the profile will be used. This is required for most structural packages to create the section.")]
```

Comment:
"If null or empty the name of the profile will be used." 

If this is true for all SectionProperty (SteelSection, ConcreteSection etc.) I think it should be included with "This is required for most structural packages..." in all name fields.

Reply:
Should be on all sections creats that explicitly accepts a profile I think. For others, you will need to provide the name, as the profile is created by the method, without a name.

Was missing in one place (for the dispatcher method), have added it there!